### PR TITLE
dbt: Add basic skeleton from `dbt init`, using `dbt-postgres`

### PR DESCRIPTION
**Pull request by amotl**
## About
@hlcianfagna contributed relevant [dbt](https://www.getdbt.com/) `macros` overrides for CrateDB [^1] the other day, based on the canonical `dbt-postgres` [^2]. This patch intends to set the stage for a wrapped-up `dbt-cratedb2` package, to be implemented in a subsequent iteration.

[^1]: https://community.cratedb.com/t/using-dbt-with-cratedb/1566
[^2]: https://github.com/dbt-labs/dbt-postgres/

### CODE REVIEW

File: framework/dbt/basic/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
+
+### Using the starter project
+
+Try running the following commands:
+- dbt run
```
Comment (kneth) [inline framework/dbt/basic/README.md:6]:
```suggestion
- `dbt run`
```

File: framework/dbt/basic/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- dbt test
```
Comment (kneth) [inline framework/dbt/basic/README.md:7]:
```suggestion
- `dbt test`
```

**Review by kneth — APPROVED** [review]
LGTM

File: framework/dbt/basic/README.md (hunk 1 above)
Comment (amotl) [reply framework/dbt/basic/README.md]: Thanks. `dbt init` created the README file that way.

**Pull request merged**